src/js/control-bar/volume-control/volume-control.js

```diff
@@ -36,6 +36,23 @@ class VolumeControl extends Component {
         this.removeClass('vjs-hidden');
       }
     });
+
+    this.on(this.volumeBar, ['focus', 'slideractive'], () => {
```

Comment:
yeah this makes it so that you can click and drag anywhere rather than having to click and drag directly on the volume slider.